### PR TITLE
Add withdraw fee estimation to the exit view

### PR DESCRIPTION
**Pull request by elias-garcia**

### CODE REVIEW

File: src/views/transaction/components/transaction-overview/transaction-overview.view.jsx [hunk 1]
```diff
@@ -58,6 +67,12 @@ function TransactionOverview ({
     )
   }
 
+  function getEstimatedWithdrawFee () {
+    return estimatedWithdrawFeeTask.status === 'successful'
+      ? getAmountInPreferredCurrency(estimatedWithdrawFeeTask.data, preferredCurrency, fiatExchangeRates).toFixed(2)
+      : '--'
```
Comment (mmilenkovic) [inline src/views/transaction/components/transaction-overview/transaction-overview.view.jsx:73]: Do you want do have any error displayed to the user here or not?

File: src/views/transaction/components/transaction-overview/transaction-overview.view.jsx [hunk 2]
```diff
@@ -171,6 +186,21 @@ function TransactionOverview ({
                 />
                 )
           }
+          {
+            transactionType === TxType.Exit && (
+              <div className={classes.exitInfoWrapper}>
+                <div className={classes.exitHelperTextWrapper}>
+                  <InfoIcon className={classes.exitHelperIcon} />
+                  <p className={classes.exitHelperText}>
+                    This step is not revertible. Once initiated, the withdrawal process must be completed.
```
Comment (mmilenkovic) [inline src/views/transaction/components/transaction-overview/transaction-overview.view.jsx:195]: revertible  -> reversible

Comment (elias-garcia) [reply src/views/transaction/components/transaction-overview/transaction-overview.view.jsx:195]: Both are valid, I think that I should ask Jenna which one is better!

File: src/views/transaction/components/transaction-overview/transaction-overview.view.jsx (hunk 1 above)
Comment (elias-garcia) [reply src/views/transaction/components/transaction-overview/transaction-overview.view.jsx:73]: Nope!

**Review by mmilenkovic — APPROVED** [review]

**Pull request merged**